### PR TITLE
Feature/views and navigation

### PR DESCRIPTION
**Pull request by esotrope**

### CODE REVIEW

File: src/main/webapp/src/assets/i18n/en.json [hunk 1]
```diff
@@ -1,27 +1,112 @@
...
     "types": [
       {
         "shortName": "SUMMATIVE",
-        "name": "Summative"
+        "name": "Summative",
+        "performanceLevels": [
+          "Level 1",
+          "Level 2",
+          "Level 3",
+          "Level 4"
```
Comment (agorina) [inline src/main/webapp/src/assets/i18n/en.json]: Is this temporary until we are done with the real work?
I think the back end needs to drive this data.

File: src/main/webapp/src/app/students/students.component.ts [hunk 2]
```diff
@@ -14,14 +14,15 @@ export class StudentsComponent implements OnInit {
...
+  constructor(private service: DataService, private route: ActivatedRoute) {
+  }
 
   ngOnInit() {
     this.route.params
       .subscribe(params => {
         this.service.getGroup(params['groupId'])
           .subscribe((group: any) => {
-              group.students = sortOn(group.students, student => student.lastName, student => student.firstName);
+              group.students = sortAscOn(group.students, student => student.lastName, student => student.firstName);
```
Comment (agorina) [inline src/main/webapp/src/app/students/students.component.ts]: Does this expect all students in a Group to be returned at once?

File: src/main/webapp/src/assets/i18n/en.json (hunk 1 above)
Comment (esotrope) [inline src/main/webapp/src/assets/i18n/en.json]: This is the local file for standalone mode. so the client can be stood up without the backend.
In "backend mode" the client hits /api/translations/{locale} for the JSON translations which can then be driven by a file + db entries

File: src/main/webapp/src/app/students/students.component.ts (hunk 2 above)
Comment (esotrope) [inline src/main/webapp/src/app/students/students.component.ts]: right now yes - will change if we expect to have large amounts of students in a group

**Pull request merged**